tools/src/versioning/ios/versionVendoredModules.ts

```diff
@@ -122,6 +122,10 @@ function baseTransformsFactory(prefix: string): Required<FileTransforms> {
         find: /react::/g,
         replaceWith: `${prefix}React::`,
       },
+      {
+        find: /hermes::/g,
+        replaceWith: `${prefix}hermes::`,
+      },
       {
         find: /namespace (facebook|react)/g,
```

Comment:
I know we're prebuilding hermes, but maybe let's add it here just in case
```suggestion
        find: /namespace (facebook|react|hermes)/g,
```